sql/pgwire/v3.go

```diff
@@ -172,6 +172,7 @@ func (c *v3Conn) serve(authenticationHook func(string, bool) error) error {
 	for key, value := range map[string]string{
 		"client_encoding": "UTF8",
 		"DateStyle":       "ISO",
+		"server_version":  "9.1.0",
```

Comment:
@mberhault ran into a problem the other day in which some piece of software required a `server_version`. I was hoping we could just put some dummy value there instead of pretending to be a specific version. This definitely needs a comment about why this particular version was chosen.